jinahub/loaders/AudioLoader/executor/audio_loader.py

```diff
@@ -0,0 +1,59 @@
+from typing import Dict, Iterable
+
+import librosa
+from jina import DocumentArray, Executor, requests
+
+
+class AudioLoader(Executor):
+    """AudioLoader loads audio file into the Document buffer."""
+
+    def __init__(
+        self,
+        audio_types: Iterable[str] = None,
```

Comment:
I don't see the necessity to have this argument. This can be infered from the the `uri`.